test/unit/classes.test.node.js

```diff
@@ -1,4 +1,4 @@
-const classes = require('./classes');
+const classes = require('../../src/util/classes');
```

Comment:
I'm not sure how aliases works in tests, but I think this could be replaced with `~/util/classes`.

Reply:
Thanks, I made the changes